content/english/github-basics/activity-3.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Adding Files to Your GitHub Repository"
+date: 2021-09-10T13:24:17-07:00
+draft: false
+weight: 4
+---
+
+Congratulations, brave coders! You've successfully set up your GitHub repository – the headquarters for your coding expedition with Maria and Kyle. 🎉🏰
+
+Now, it's time to stock your repository with precious files and enchanted code! Let's learn the two ways to add files to it! ✨📂
+
+First, go into your new repository, click on the "Add file" button, and select "Create new file".
+- Now you can name your new file and start coding! Below is a photo of a file created by Maria to say hello to Kyle:
+
+<img alt="Creating a new file in the GitHub repository named hello_world.py and adding code to it" src="../images/NewFile.png" height="400"/>
+
+- After you've finish coding, scroll down to the bottom of the page and you will see this:
+<img alt="Commit options for the new file" src="../images/commit_new.PNG" height="400"/>
+
+<img alt="GitHub's repository view with hello_world.py showing in the list of files" src="../images/tempsnip4.png" height="400"/>
+
+<img alt="File upload options in the GitHub repository" src="../images/UploadNewFile.png" height="400"/>
+
+# Your GitHub Repository Blooms! 🌱🏞️
```

Comment:
Due to accessibility, we can't use # (h1) for any of the headers, all headers must be at least ## (h2)

```suggestion
## Your GitHub Repository Blooms! 🌱🏞️
```